timetracker/vms/views.py

```diff
@@ -403,7 +403,17 @@ def get_context_data(self, **kwargs):
             client__slug=self.kwargs.get('client_slug'),
             employee_id=self.kwargs.get('employee_id'),
         )
+        open_time_record = None
+        if employee.time_records.filter(time_end=None).first():
+            open_time_record = employee.time_records.filter(
+                time_end=None,
+            ).first()
```

Comment:
This should just be

```python
open_time_record = employee.time_records.filter(time_end=None).first()
```

Otherwise we're hitting the database twice for no reason, and `first()` returns `None` if there is nothing in the queryset anyway, so we're not gaining anything from the if statement.